shell/platform/android/io/flutter/app/FlutterActivityDelegate.java

```diff
@@ -43,7 +43,7 @@
  * The most obvious example of when this may come in handy is if an application wishes to subclass
  * the Android v4 support library's {@code FragmentActivity}.
  *
- * <p><b>Usage:</b></p>
```

Comment:
Any reason why the closing </p> was removed?

Reply:
The closing tag `</p>` isn't required.  See this thread https://github.com/google/google-java-format/issues/61